### ISSUE TITLE
April141979!

### ISSUE THREAD

**Issue by magdeline401-afk** [auto-wcag/auto-wcag#2426]
## Accessibility support

(( Put the accessibility support item here ))

## ACT Rules

- ... List the rules here, link to its acc support section

### Example 1 ...

(( Add example here ))

```html
code snippet here
```

[Open this example]()

(( Add a link to a code pen or other hosted example. Keep in mind this example needs to be accessible and persistent. ))

#### Preconditions

Before running the test, check your settings. In many cases default settings aren't strictly required and will not impact the results. Use common sense here. If non-default settings are necessary for you, record the settings used in the reported results.

- Ensure browsers are in the default settings, turn off unnecessary extensions
- Ensure ATs are in the default settings
- Ensure operating system are in default settings
- Ensure software uses the correct version (usually the latest version)

#### Test instructions

(( Provide instructions on how to test the example, this should include: ))

- Which technologies to use
- Any setting changes necessary to run the test
- Any variation in how to operate the different assistive technologies
- Step by step instructions on how to check the example
- What the expected results are

### External links

(( List any external links to bugs in browsers & AT, WPT, or spec issues. Be sure to open issues in the appropriate repositories. ))

## Test results

Please report test results in a comment below. Please use the following format:

```md
## Test case X

- Date: ...
- Operating system: ..., version ...
- Browser: ..., version ...
- Assistive technology: ..., version ...
- Test result: Passed / Failed (if failed, how was it different from expected?)
```

**Comment by WilcoFiers** [comment]
@magdeline401-afk This looks like an accidentally opened issue. Closing it. Was there anything you were trying to do here?
